src/data/sfsgoals.cfg

```diff
@@ -15,15 +15,45 @@
 # Custom goals can be defined with or without labels.
 #
 # EXAMPLES:
-#
-#  3 3 : _ _ _                                 # one of the default goals (three copies anywhere)
-#  8 not_important_file : _                    # only one copy
+# One of the default goals (three copies anywhere)
+#  3 3 : _ _ _
+
+# Only one copy
+#  8 not_important_file : _
+
+# Two copies
 # 11 important_file : _ _
-# 12 local_copy_on_mars : mars _               # at least one copy in the Martian datacenter
-# 13 cached_on_ssd : ssd _
-# 14 very_important_file : _ _ _ _
-# 15 min_two_locations: _ locationA locationB  # one copy in A, one in B, third anywhere
-# 16 fast_access      : ssd _ _                # one copy on ssd, two additional on any drives
-# 17 two_manufacturers: WD HT                  # one on WD disk, one on HT disk
-# 18 default_ec : $ec(5,2)                     # erasure code with 5 data parts and 2 parity parts
-# 19 wide_ec : $ec(17,9) { ssd ssd }           # erasure code 17+9 with at least two parts on ssd
+
+# Four copies
+# 12 very_important_file : _ _ _ _
+
+# At least one copy one the chunkserver labeled 'mars', and second copy
+# anywhere (including any other chunkserver labelled mars)
```

Comment:
There are some places with 'labelled' and others with 'labeled', even if they are both accepted, please consider unifying it.

Reply:
Went with labeled since that was predominant already. I thin generally we should stick with U.S spelling, since that's very common in technical documentations.